### ISSUE TITLE
Bridge for ASE trajectory files

### ISSUE THREAD

**Issue by schneiderfelipe** [cclib/cclib#843]
I would like to build a bridge from cclib to the [Atomic Simulation Environment (ASE)](https://wiki.fysik.dtu.dk/ase/index.html). That would consist of a small module (`cclib.cclib.bridge.cclib2ase`) with support of creating `ase.Atoms` objects and writing ASE trajectory files. ASE would be kept as a soft dependency, of course.

`ase.Atoms` objects hold only structural data (atomic numbers, masses, geometries, etc.), but [trajectory files](https://wiki.fysik.dtu.dk/ase/ase/io/trajectory.html) can store a [small set of calculated properties](https://gitlab.com/ase/ase/-/blob/master/ase/calculators/calculator.py#L118), such as energies, gradients, dipoles, charges, and free energies. Those properties would be taken from cclib whenever available.

Such a bridge would make cclib interoperable with any package expecting ASE trajectory files as data sources. For example, one could directly use data read with cclib (e.g., geometries, energies, and gradients) to train neural networks for fast energy and gradient predictions, as the [Atomistic Machine-learning Package (Amp)](https://amp.readthedocs.io/en/latest/) expects train data in the ASE trajectory format.

I would be happy to submit a PR if you believe that it is something interesting/useful.

**Comment by berquist** [comment]
Oops, fat fingered that one...yes, that would be a really nice feature to have.

**Comment by langner** [comment]
Indeed, this would be nice.

**Comment by berquist** [comment]
Done in #845
